flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/source/utils/StatementUtils.java

```diff
@@ -131,8 +131,12 @@ public static Object queryNextChunkMax(
     }
 
     public static String buildSplitScanQuery(
```

Comment:
Maybe we can add a test for the method as there is not test here before.

Reply:
done